wandb: WARNING Calling wandb.login() after wandb.init() has no effect.
Create sweep with ID: nadh9sgz
usage: run_graphmimic.py [-h] --data_dir DATA_DIR --model_name_or_path MODEL_NAME_OR_PATH --output_dir OUTPUT_DIR [--config_name CONFIG_NAME] [--tokenizer_name TOKENIZER_NAME] [--cache_dir CACHE_DIR]
                         [--task_name TASK_NAME] [--max_seq_length MAX_SEQ_LENGTH] [--do_train] [--do_eval] [--evaluate_during_training] [--do_lower_case] [--per_gpu_train_batch_size PER_GPU_TRAIN_BATCH_SIZE]
                         [--per_gpu_eval_batch_size PER_GPU_EVAL_BATCH_SIZE] [--gradient_accumulation_steps GRADIENT_ACCUMULATION_STEPS] [--learning_rate LEARNING_RATE] [--weight_decay WEIGHT_DECAY]
                         [--adam_epsilon ADAM_EPSILON] [--max_grad_norm MAX_GRAD_NORM] [--num_train_epochs NUM_TRAIN_EPOCHS] [--max_steps MAX_STEPS] [--warmup_steps WARMUP_STEPS] [--logging_steps LOGGING_STEPS]
                         [--save_steps SAVE_STEPS] [--eval_steps EVAL_STEPS] [--eval_all_checkpoints] [--no_cuda] [--overwrite_output_dir] [--overwrite_cache] [--seed SEED] [--fp16] [--fp16_opt_level FP16_OPT_LEVEL]
run_graphmimic.py: error: the following arguments are required: --data_dir, --model_name_or_path, --output_dir
Sweep URL: https://wandb.ai/dc925/graphmimic/sweeps/nadh9sgz
